Portfolio/index.css

```diff
@@ -1 +1,237 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+    margin: 0;
+    padding: 0;
+}

```

Comment:
The body selector should have some properties defined within the curly braces for better formatting.